content/vocabularies/g-sgov-slovník-pro-agendu-a824-patenty/g-sgov-slovník-pro-agendu-a824-patenty-model.ttl

```diff
@@ -0,0 +1,11056 @@
+@prefix a-popis-dat-pojem: <http://onto.fel.cvut.cz/ontologies/slovník/agendový/popis-dat/pojem/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix g-sgov-slovník-pro-agendu-a824-patenty: <https://slovník.gov.cz/generický/slovník-pro-agendu-a824-patenty/> .
+@prefix g-sgov-slovník-pro-agendu-a824-patenty-pojem: <https://slovník.gov.cz/generický/slovník-pro-agendu-a824-patenty/pojem/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+g-sgov-slovník-pro-agendu-a824-patenty:model a a-popis-dat-pojem:model, owl:Ontology;
+  dcterms:created "2024-02-14T11:14:03.818Z"^^xsd:dateTime;
+  dcterms:rights <https://creativecommons.org/licenses/by-sa/4.0>;
+  dcterms:title "Slovník pro agendu A824 Patenty - model"@cs;
+  vann:preferredNamespacePrefix "g-sgov-slovník-pro-agendu-a824-patenty-pojem";
+  vann:preferredNamespaceUri "https://slovník.gov.cz/generický/slovník-pro-agendu-a824-patenty/pojem/";
+  owl:imports g-sgov-slovník-pro-agendu-a824-patenty:glosář;
+  owl:versionIRI "https://slovník.gov.cz/generický/slovník-pro-agendu-a824-patenty/model/verze/1.0.0" .
+
+<https://slovník.gov.cz/datový/pracovní-prostor/pojem/přílohový-kontext/instance-1e128290-c0b3-4d6a-b28b-14ee38733752>
+  a <https://slovník.gov.cz/datový/pracovní-prostor/pojem/přílohový-kontext>;
+  a-popis-dat-pojem:má-typ-přílohy <http://onto.fel.cvut.cz/ontologies/application/ontoGrapher/diagram>;
+  <https://slovník.gov.cz/datový/pracovní-prostor/pojem/vychází-z-verze> <https://slovník.gov.cz/příloha/instance-1e128290-c0b3-4d6a-b28b-14ee38733752> .
+
+g-sgov-slovník-pro-agendu-a824-patenty-pojem:přihláška-vynálezu a <https://slovník.gov.cz/základní/pojem/typ-objektu>;
```

Comment:
vazba subClassOf se používá jako dědičnost, tzn. že přihláška vynálezu je zároveň patentovým spisem a může mít všechny atributy a vazby které má patentový spis. Předpokládám, že to asi spíše není pravda

Reply:
> doporučil bych vytvořit novou vazbu, např. je má přihlášku vynálezu